packages/atomic/cypress/integration/result-list/result-components/result-multi-value-text.cypress.ts

```diff
@@ -0,0 +1,314 @@
+  field?: string | number;
+  'max-values-to-display'?: number;
+}
+
+const addMultiValueText = (
+  props: MultiValueTextProps = {},
+  slot: HTMLElement[] = []
+) => {
+  const multiValueTextEl = generateComponentHTML(
+    resultMultiValueTextComponent,
+    props as TagProps
+  );
+  slot.forEach((el) => multiValueTextEl.appendChild(el));
+  return addResultList(
+    buildTemplateWithSections({bottomMetadata: multiValueTextEl})
+  );
+};
+
+export interface FacetWithResponseProps {
+  facetId: string;
+  field: string;
+  values: string[];
+  selectedValues: string[];
+}
+
+const addFacetWithResponse =
+  ({facetId, field, values, selectedValues}: FacetWithResponseProps) =>
+  (fixture: TestFixture) => {
+    fixture
+      .with(addFacet({'facet-id': facetId, field}))
+      .withCustomResponse((response) => {
+        response.facets = [
+          {
+            facetId,
+            field,
+            indexScore: 0,
+            moreValuesAvailable: false,
+            values: values.map((value) => ({
+              value,
+              numberOfResults: 1337,
+              state: selectedValues.includes(value) ? 'selected' : 'idle',
+            })),
+          },
+        ];
+      });
+  };
+
+describe('Result MultiValueText Component', () => {
+  describe('when not used inside a result template', () => {
+    beforeEach(() => {
+      new TestFixture()
+        .withElement(generateComponentHTML(resultMultiValueTextComponent))
+        .init();
+    });
+
+    it.skip('should remove the component from the DOM', () => {
+      cy.get(resultMultiValueTextComponent).should('not.exist');
+    });
+
+    it.skip('should log a console error', () => {
+      cy.get(resultMultiValueTextComponent)
+        .shadow()
+        .find('atomic-component-error')
+        .should('exist');
+    });
+  });
+
+  describe('when used inside a result template', () => {
+    describe('when the field does not exist for the result', () => {
+      beforeEach(() => {
+        new TestFixture()
+          .with(addMultiValueText({field: 'thisfielddoesnotexist'}))
+          .init();
+      });
+
+      it.skip('should remove the component from the DOM', () => {
+        ResultMultiValueTextSelectors.firstInResult().should('not.exist');
+      });
+    });
+
+    describe('when the field value is not a string nor a string array', () => {
+      beforeEach(() => {
+        new TestFixture()
+          .with(addMultiValueText({field: 420}))
+          .withCustomResponse((response) => {
+            response.results.forEach((result) => (result.raw['420'] = 'Abc'));
+          })
+          .init();
+      });
+
+      it.skip('should remove the component from the DOM', () => {
+        ResultMultiValueTextSelectors.firstInResult().should('not.exist');
+      });
+    });
+
+    describe('when the field value exists & is a string array', () => {
+      const field = 'hello_world';
+      function prepareValuesWithMaximum(
```

Comment:
I'm not sure about the name "prepare" here. This doesn't initialize the fixture (unlike the `setup` functions). A possible alternative would be to make this a callback/testfeature.